lib/tasks/db.js

```diff
@@ -401,3 +451,26 @@ function tryToMigrateFromOldSchema () {
     });
   });
 }
+
+/**
+ * ensureSeeds - checks that the `--seed` option exists
+ */
+
+function ensureSeeds () {
+  if ( !args.seed ) {
+    helpers.view.error(
+      'Unspecified flag ' +
+      clc.blueBright('"seed"') +
+      '. Check the manual for further details.'
+    );
+    process.exit(1);
+  }
+
+  if ( !_.isArray(args.seed) ) {
```

Comment:
Btw. how would you end up having an array here? If you call the CLI with `--seed`  twice?